client/css/style.css

```diff
@@ -1915,6 +1916,16 @@ kbd {
 		padding: 0;
 	}
 
+	#chat .message .from .user:before {
```

Comment:
It actually has nothing to do with colors, and because it's shoved in the middle of other changes/commits, I didn't see it until now.

I don't think we want this. I'm not too much in favor, but I would need to play with it to see how it feels / test usability. In the meantime, @bews please remove it or move it to a dedicated PR.